Clean Swift/Worker.xctemplate/___FILEBASENAME___Worker.swift

```diff
@@ -3,7 +3,7 @@
 //  ___PROJECTNAME___
```

Comment:
Since worker may contain additional/shared `func`s that may extend beyond use cases, can we have the template in the following format?

```Swift
class xyzWorker {

    // MARK: - Properties
    ...

    // MARK: - Methods
    ...
}
```

(This would be the new standard for upcoming workers as well)

- [ ] Follow the above format

Reply:
OK